app/src/ui/lib/text-box.tsx

```diff
@@ -118,11 +115,6 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
 
   private onChange = (event: React.FormEvent<HTMLInputElement>) => {
     const value = event.currentTarget.value
-
-    if (this.props.onChange) {
-      this.props.onChange(event)
-    }
-
     const defaultPrevented = event.defaultPrevented
```

Comment:
I think I'm into this change, especially if we can't think of a scenario where we want to block the text changing, but this line here should also be :fire:'d (and maybe some others, I haven't read this whole function closely....

Reply:
Oh yes, you're right! ⚡️